### PR TITLE
create Text.php for zh_CN

### PR DESCRIPTION
**Pull request by sandysong**
create Text.php for zh_CN, since 'zh_CN' and 'zh_TW' are similar to each other, so the code is just copy from zh_TW

### CODE REVIEW

**Comment by fzaninotto** [comment]
Copying code is bad. Why don't you use inheritance?

**Comment by seekcx** [comment]
Simplified and traditional, although the same content, but the characters are not exactly the same, so can not simply use inheritance to solve the problem.

I am also zh-cn no text support and distress, see the commit, should be a solution, hoping to merge.

**Comment by fzaninotto** [comment]
I don't understand your answer, can you elaborate?

**Comment by fzaninotto** [comment]
No feedback for a month, closing
